HtmlCssSample.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="HtmlCssSample.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Sample Form</title>
+</head>
+
+<body>
+    <nav>
+        <picture class="logo">
+            <img src="./images.png" alt="site logo">
+        </picture>
+        <div>
+            <p>Pricing</p>
+            <p>Solutions</p>
+            <p>Community</p>
+            <p>Resources</p>
+            <p>Contact</p>
+            <section class="buttons">
+                <button class="btn-sign-in">Sign in</button>
+                <button class="btn-register">Register</button>
+            </section>
+
+        </div>
+    </nav>
+    <main>
+        <header>
+            <div class="html-heading">
+                HTML
+            </div>
+            <div class="css-heading">
+                CSS
+            </div>
+        </header>
+        <section class="form-body">
+            <form>
+                <label for="Name">Name</label>
+
```

Comment:
Use a more semantic structure for the form, such as a **fieldset** element, to group related form elements.

Reply:
Resolved